rcljava/src/test/java/org/ros2/rcljava/SpinTest.java

```diff
@@ -333,4 +343,62 @@ public Node getNode() {
     executor.spinAll(100*1000*1000);
     assertEquals(1, timerCallback.getCounter());
   }
+
+  // custom event consumer
+  public static class EventConsumer implements Consumer<OfferedQosIncompatible> {
+    public boolean done = false;
+
+    public void accept(final OfferedQosIncompatible status) {
+      assertEquals(status.totalCount, 1);
+      assertEquals(status.totalCountChange, 1);
+      assertEquals(status.lastPolicyKind, OfferedQosIncompatible.PolicyKind.RELIABILITY);
+      this.done = true;
+    }
+  }
+
+  @Test
+  public final void testSpinEvent() {
+    String identifier = RCLJava.getRMWIdentifier();
+    if (identifier == "rmw_fastrtps_cpp" || identifier == "rmw_fastrtps_dynamic_cpp") {
+      // OfferedQosIncompatible event not supported in these implementations
+      return;
+    }
+    System.out.print(identifier);
```

Comment:
I think we can remove this print statement.

Reply:
I forgot I added that print for debugging, but I never fixed the issue :joy:.
Fixed in https://github.com/osrf/ros2_java/pull/11/commits/ce1ba9f1708c2f35713e19892f77977049f6ccfe.